### PR TITLE
[Cloud Sec - Administration] cna fixes and updates

### PR DESCRIPTION
**Pull request by arane26**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/administration/network-security/cloud-network-analyzer.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+== Cloud Network Analyzer
+
+//Overview of cloud network analyzer.
+
+Prisma Cloud’s cloud network analyzer (CNA) takes a multi-dimensional approach to identify overly-exposed assets by providing an end-to-end network path visibility from any source, such as AWS EC2 virtual machine, DB instance, or Lambda application to any destination, such as the Internet, another VPC, or on-premises networks. For example, a compute instance is directly exposed to the Internet only when it has an elastic network interface (ENI) with EIP attached to a public subnet and has overly permissive security-group in a VPC that is attached to an Internet gateway with route to the Internet.
```
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/administration/network-security/cloud-network-analyzer.adoc:5]: Should Cloud Network Analyzer (CNA) be capitalized here?

File: docs/en/enterprise-edition/content-collections/administration/network-security/network-security.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+====
+Network exposure queries are currently supported only on AWS, Azure, and GCP cloud environments and are currently not available in the Government and China regions.
+====
+
+Prisma Cloud’s inbuilt cloud network analyzer (CNA) engine automatically calculates net effective reachability of cloud assets through entities such as virtual machines, databases, and network interfaces and helps detect unrestricted network access from the Internet or external network domain using two main vectors:
+
+* Routing path exists from source to destination
+* Net effectiveness of all cloud-native network security policies in the path
+
+Prisma Cloud network security capabilities include high fidelity alerts that provide rich context, so you know exactly how a particular cloud asset is exposed and can prioritize the risk and take meaningful action. It includes out-of-the-box policies that help you identify risky network exposure.
```
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/administration/network-security/network-security.adoc:19]: I'd suggest leading with the user benefit in this sentence:

Identify exactly how a particular cloud asset is exposed, prioritize risk and take meaningful action with Prisma Cloud's network security capabilities including high fidelity alerts that provide rich security context.

**Review by jenjoe22 — APPROVED** [review]
Provided a few suggested edits but otherwise looks good to me.

**Comment by arane26** [comment]
incorporated Jennifer's review comment

**Pull request merged**